recipes/libsemigroups/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "libsemigroups" %}
+{% set version = "0.2.1" %}
+{% set md5 = "de073733a7d389e6ce378f4586ff1f42" %}
+{% set repo = "https://github.com/james-d-mitchell/libsemigroups" %}
+{% set home = "https://james-d-mitchell.github.io/libsemigroups/" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: {{ repo }}/releases/download/v{{ version }}/{{ name }}-{{ version }}.tar.gz
+  md5: {{ md5 }}
+  #git_rev: v{{ version }}
+  #git_url: {{ repo }}.git
+  #git_tag: autotools
+
+build:
+  number: 0
+
+test:
+  commands:
+    - test -d ${PREFIX}/include/libsemigroups  # [unix]
+    - test -f ${PREFIX}/include/libsemigroups/semigroups.h  # [unix]
+    - if exist %LIBRARY_PREFIX%\include\libsemigroups\semigroups.h (exit 0) else (exit 1)  # [win]
+
+about:
+  home: {{ home }}
+  license: GPL-3.0
+  #license_file: LICENSE # temporary workaround: the LICENSE file is missing in the tarbal
+  summary: 'C++ library for semigroups and monoids'
+
+  description: |
+    libsemigroups is a C++ library for semigroups and monoids; it is partly based on 
+    [Algorithms for computing finite semigroups](https://www.irif.fr/~jep/PDF/Rio.pdf), 
+    [Expository Slides](https://www.irif.fr/~jep/PDF/Exposes/StAndrews.pdf), and 
+    [Semigroupe 2.01](https://www.irif.fr/~jep/Logiciels/Semigroupe2.0/semigroupe2.html) by Jean-Eric Pin.
+  doc_url: {{ home }}
+  dev_url: {{ repo }}
+
+extra:
+  recipe-maintainers:
+    - james-d-mitchell
```

Comment:
Did you ask `james-d-mitchell` whether he wants to be a maintainer for this conda package?